webcompat/webhooks/helpers.py

```diff
@@ -59,36 +63,41 @@ def extract_browser_label(body):
         return None
 
 
-def extract_media_info(body):
+def get_issue_info(payload):
+    """Extract all information we need when handling webhooks for issues."""
+    # Extract the title and the body
+    title = payload.get('issue')['title']
+    body = payload.get('issue')['body']
+    # Create the issue dictionary
+    return {'action': payload.get('action'),
+            'number': payload.get('issue')['number'],
+            'domain': title.partition(' ')[0],
```

Comment:
`str.partition('char')` and `str.rpartition`, `str.split('char', steps)` and str.rsplit() are all very useful and often faster than regex. See in the comments of issue #1551